### PR TITLE
enable hyperv for ceilometer agent role

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: crowbar_framework/app/models/ceilometer_service.rb [hunk 1]
```diff
@@ -33,9 +33,7 @@ def role_constraints
         "ceilometer-agent" => {
           "unique" => false,
           "count" => -1,
-          "exclude_platform" => {
-            "windows" => "/.*/"
-          }
+          "exclude_platform" => {}
```
Comment (tboerger) [inline crowbar_framework/app/models/ceilometer_service.rb]: Please just remove that if there are no constraints.

**Comment by tboerger** [comment]
Small nitpick but i'm fine with that :+1:

File: crowbar_framework/app/models/ceilometer_service.rb (hunk 1 above)
Comment (jsuchome) [inline crowbar_framework/app/models/ceilometer_service.rb]: Hm, I like it more with empty set :-)

**Comment by vuntz** [comment]
Btw, it's not ready for merging until the hyperv barclamp is updated accordingly

File: crowbar_framework/app/models/ceilometer_service.rb (hunk 1 above)
Comment (tboerger) [inline crowbar_framework/app/models/ceilometer_service.rb]: So go ahead and add it to ALL barclamps and roles please. If you don't want to do that remove it :)

**Comment by mjura** [comment]
Should we add here dependency to PR which is changing barclamp-hyperv and allow to deploy ceilometer-agent on hyperv node

**Comment by jsuchome** [comment]
Obsoleted by @nkrinner's changes

**Pull request closed without merging**